components/menu/menu-item-mixin.js

```diff
@@ -24,6 +24,11 @@ export const MenuItemMixin = superclass => class extends superclass {
 			 * @ignore
 			 */
 			last: { type: String, reflect: true }, // set by d2l-menu
+			/**
+			 * The number of lines to display before truncating text with an ellipsis. Defaults to 2.
+			 * @type {number}
+			 */
+			lines: { type: Number },
```

Comment:
Set the default in the constructor?

Reply:
Was trying to avoid calling `setProperty('--d2l-menu-item-lines', this.lines)` unnecessarily, given that the default is in the CSS. But as it stands, anything that relies on `this.lines` would get `undefined`, which isn't great.

Reply:
Just to close the loop - I added this to the constructor along with a check so we don't call `setProperty` it unnecessarily.